git_machete/tests/functional/test_machete.py

```diff
@@ -43,7 +43,7 @@ def __init__(self) -> None:
 class MockGithubAPIState:
     def __init__(self, pulls: List[Dict[str, Any]]) -> None:
         self.pulls: List[Dict[str, Any]] = pulls
-        self.user: Dict[str, str] = {'login': 'github_user', 'type': 'User', 'company': 'VirtusLab'}
+        self.user: Dict[str, str] = {'login': 'github_admin', 'type': 'User', 'company': 'VirtusLab'}
```

Comment:
Needed to change user, cause with the same one, author of pull request is no added to annotations

Reply:
Okay! maybe use sth like `other_user` to mark the intent more clearly